### ISSUE TITLE
Update UserGuide?

### ISSUE THREAD

**Issue by matthiasballreich** [xdanaux/moderncv#25]
Can you please update the UserGuide. The one are here on Github are outdated and for new users - like me, an new UserGuide would be much helpful. thanks

**Comment by jayree** [comment]
Have you looked into the updated user guide: [moderncv_userguide.v2](https://github.com/xdanaux/moderncv/blob/master/manual/moderncv_userguide.v2.pdf)

**Comment by qin-nz** [comment]
Maybe you should put the link <moderncv_userguide.v2> in README.md

**Comment by matthiasballreich** [comment]
yes i looked into userguideV2. But this one if for version 0.6 of moderncv and now moderncv has version 2.0. So maybe some things have changed or some new better / easier things where added and these should maybe described there.
